ams/server/src/test/java/com/netease/arctic/server/table/executor/TestOrphanFileCleanIceberg.java

```diff
@@ -42,4 +71,69 @@ public TestOrphanFileCleanIceberg(CatalogTestHelper catalogTestHelper, TableTest
     super(catalogTestHelper, tableTestHelper);
   }
 
+  @Test
+  public void cleanIndependentFiles() throws IOException {
+    List<Record> records = Lists.newArrayListWithCapacity(3);
+    records.add(tableTestHelper().generateTestRecord(1, "test1", 0, "2022-01-01T00:00:00"));
+    List<DataFile> dataFiles1 = tableTestHelper().writeBaseStore(getArcticTable().asUnkeyedTable(), 1L,
+        records, false);
+    UnkeyedTable testTable = getArcticTable().asUnkeyedTable();
+    AppendFiles appendFiles = testTable.newAppend();
+    dataFiles1.forEach(appendFiles::appendFile);
+    appendFiles.commit();
+    assertIndependentFiles(testTable, 0);
+    SortedPosDeleteWriter<Record> posDeleteWriter = AdaptHiveGenericTaskWriterBuilder.builderFor(testTable)
+        .buildBasePosDeleteWriter(0, 0, dataFiles1.get(0).partition());
+    posDeleteWriter.delete(dataFiles1.get(0).path(), 0);
+    List<DeleteFile> posDelete = posDeleteWriter.complete();
+    testTable.newRowDelta().addDeletes(posDelete.get(0)).commit();
+    records.clear();
+    records.add(tableTestHelper().generateTestRecord(3, "test3", 0, "2022-01-02T00:00:00"));
+    List<DataFile> dataFiles2 = tableTestHelper().writeBaseStore(getArcticTable().asUnkeyedTable(), 1L,
+        records, false);
+    appendFiles = testTable.newAppend();
+    dataFiles2.forEach(appendFiles::appendFile);
```

Comment:
The `appendFiles` is not committed, so it didn't have any effect. I think we can delete these two lines.